src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2855,6 +2855,11 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
                 ->addArgument(new Reference('http_client', ContainerBuilder::NULL_ON_INVALID_REFERENCE));
         }
 
+        if (ContainerBuilder::willBeAvailable('symfony/bluesky-notifier', NotifierBridge\Bluesky\BlueskyTransportFactory::class, ['symfony/framework-bundle', 'symfony/notifier'])) {
+            $container->getDefinition($classToServices[NotifierBridge\Bluesky\BlueskyTransportFactory::class])
+                ->addArgument(new Reference('logger'));
```

Comment:
IMO this should be done in the service definition directly (in the config file), not here.

Reply:
At first I was on that track as well, but then I realised that we do that for other services (see the modifications done for Mercue, FakeChat and FakeSms above)  as well since we use a rather generic approach to register notifier transport factory services since #53222. I didn't want to start mixing approaches to register additional arguments.